setup.sh

```diff
@@ -8,31 +8,49 @@
 
 # Follow 'JULES from scratch' manual to install Cylc, Rose and FCM and cache your morsr password
 
-# Install Cylc
+## WB: seems like we need python2, so let's create a virtualenv first:
+
+wget https://repo.anaconda.com/archive/Anaconda3-2022.05-Linux-x86_64.sh
+bash Anaconda3-2020.02-Linux-x86_64.sh
+
+conda install -c conda-forge pygraphviz # this solved some error messages
+
+# make sure you copy the file environment.yml from the git repo
+# note that we have two environments, one for jules and one for mosart!
+
+conda env create -f environment-jules.yml
+
+# Once donwload complete, run
+conda activate jules
+
+# Install Cylc    ## WB: can we use package cylc-flow?
 mkdir ~/.local
 cd ~/.local
+
 git clone https://github.com/cylc/cylc.git
 
 cd cylc
-git tag -l
-git checkout tags/t.6.0
+#git tag -l
+git checkout tags/6.0.2
 cd ..
+export PATH=$HOME/.local/cylc/bin:$PATH
 cylc --version
 cd ~/.local/cylc
-make
+make                            # WB: is install needed?? What does this do? Getting latex compilation errors
 cd ../..
 
-export PATH =$HOME/.local/cylc/bin:$PATH
-.~/.bashrc
+export PATH=$HOME/.local/bin:$PATH
+.~/.bashrc                      # WB: what is this for?
```

Comment:
run your .bashrc file or log off and log in again to append the Cylc command set to your library